weather/weather_forecast.py

```diff
@@ -19,68 +18,21 @@ def retrieve_data_about_the_weather_forecast():
     """
     weather_forecast_text = ''
 
-    BASE_URL = "http://api.openweathermap.org/data/2.5/weather?"
-    CITY = "Sofia"
+    base_url = "http://api.openweathermap.org/data/2.5/weather?"
+    city = "Sofia"
 
-    url = BASE_URL + "q=" + CITY + "&appid=" + "f4e5b648421e4d481828851a5763cbf1"
+    url = base_url + "q=" + city + "&appid=" + "f4e5b648421e4d481828851a5763cbf1"
```

Comment:
Try to implement `try - except` statement IF city not found --> Dancho should inform the user

Reply:
Fixed with 44c51aee3a67b83a0086dce4c0cf4081f91466d4